### PR TITLE
Code cleanup, styling fixes

### PR DESCRIPTION
**Pull request by sharkmu**
This PR removes unnecessary parts, also fixes some basic errors.

### CODE REVIEW

File: src/popoverManager.ts [hunk 1]
```diff
@@ -67,12 +67,12 @@ export class PopoverManager<T> extends Component {
         this.lastEvent = null;
     }
 
-    hide(lazy: boolean = false) {
+    hide(lazy = false) {
```
Comment (RyotaUshio) [inline src/popoverManager.ts]: I agree that we don't need explicit typing here, but I don't see why it can cause any errors. Can I ask why this change is needed? Do you see any compilation errors?

Comment (RyotaUshio) [reply src/popoverManager.ts]: It'd make sense if this is just a cleanup. But personally, I prefer writing both a type and a default value.

**Comment by RyotaUshio** [comment]
Thanks for this PR. I don't know why I messed up the code with these random semicolons, maybe I was tired haha.

File: src/popoverManager.ts (hunk 1 above)
Comment (sharkmu) [reply src/popoverManager.ts]: To be honest in my opinion it is more readable this way, also for me the IDE threw an error for it so that's why I thought I should remove it.

Comment (RyotaUshio) [reply src/popoverManager.ts]: I believe that's not an "actual" error but a styling error from an eslint rule: https://typescript-eslint.io/rules/no-inferrable-types/

Personally, I feel an explicit typing helps keep consistency with other parameter declarations such as `event: UserEvent | null = null`.
So I will update `.eslintrc` rather than changing the code:

https://github.com/RyotaUshio/obsidian-quick-preview/blob/1f2abae30592d3b8de6033e58c07741733935fc3/.eslintrc#L23

Even so, thanks for spotting that!
